lib/elb/ELB.rb

```diff
@@ -69,6 +101,48 @@ def init_elbs
 
         Hash[elbs.flatten.map { |elb| [elb.load_balancer_name, elb] }]
       end
+
+      # Internal: Load ELB tags and map them to their names
+      #
+      # Returns the ELB tags mapped to ELB name
+      def init_elb_tags
+        tags = []
+        elbs.keys.each_slice(20) do |names|
```

Comment:
Tags aren't returned by the normal describe_load_balancers.  In the case where you are doing a sync one or diff one, you are getting extra data, but doing them one at a time when you do a full sync/diff would result in one call per ELB.  I figure the common case is going to be syncing all at once (which we will be doing when we have the cumulus service running).  And of course, undocumented anywhere, is a limit of 20 elb's per request for tags which is why I slice.